chainercv/evaluations/eval_semantic_segmentation_iou.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import division
+
+import numpy as np
+
+
+def calc_semantic_segmentation_confusion(pred_labels, gt_labels, n_class):
+    """Collect confusion matrix.
+
+    Args:
+        pred_label (iterable of numpy.ndarray): A collection of predicted
+            labels. The shape of a label array
+            is :math:`(H, W)`. :math:`H` and :math:`W`
+            are height and width of the label.
+            For example, This is an array of shape :math:`(N, H, W)`.
```

Comment:
This example is not good because this function can take images whose shapes are different each other. `(N, H, W)` sounds that all images should have the same shape. How about `[label0, label1, ...], where label0.shape = (H0, W0), label1.shape = (H1, W1) .. `?